docs/source/reference_aggregation.rst

```diff
@@ -0,0 +1,191 @@
+Other Methods of Aggregations
+=============================
+
+As we have already seen in this `page <https://fsspec.github.io/kerchunk/test_example.html#multi-file-jsons>`_,
+that the main purpose of ``kerchunk`` it to generate references, to view whole archive of files like
+GRIB2, NetCDF etc. allowing us for *in-situ* access to the data. In this part of the documentation,
```

Comment:
in-situ -> direct
(I think in-situ here would imply that the processing is happening wherever the data is, which is not the case; I know what you mean is that we read the originals, not a copy)